pages/api/create_class_teacher.js

```diff
@@ -1,6 +1,33 @@
 import prisma from '../../prisma/prisma';
+import { unstable_getServerSession } from 'next-auth';
+import { authOptions } from './auth/[...nextauth]';
 
 export default async function handle(req, res) {
+  //unstable_getServerSession is recommended here: https://next-auth.js.org/configuration/nextjs
+  const session = await unstable_getServerSession(req, res, authOptions);
+
+  if (!req.method == 'POST') {
+    res.status(405).end();
+  }
+
+  if (!session) {
+    res.status(403).end();
+  }
+
+  let users = await prisma.user.findMany({
+    where: {
+      role: 'TEACHER'
```

Comment:
check if prisma lets you put array for ['TEACHER', 'ADMIN'] here

Reply:
Didn't need anymore. Went a new route by using findUnique query